<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Theke</class>
 <widget class="QDialog" name="Theke">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox::title{subcontrol-origin:margin; left:10px; padding: 0 3px 0 3px;color:rgb(10,10,10);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(10,10,10);</string>
   </property>
   <property name="text">
    <string>Theke Licht</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bilder/Icons/back_green.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="bilder.qrc">
      <normaloff>:/bilder/Icons/arrow_left_icon_blk.png</normaloff>:/bilder/Icons/arrow_left_icon_blk.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_RGB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>81</y>
     <width>339</width>
     <height>175</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">margin-top:0.5em;
border:1px solid black;
border-radius:10px;
font-size:20px;</string>
   </property>
   <property name="title">
    <string>RGB</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">;margin-top:0;
border:none;
background-color:rgba(0,0,0,100);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_rgb_togge">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>111</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0);
border:none;
color:rgb(255,255,255);</string>
        </property>
        <property name="text">
         <string>An</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">;margin-top:0;
border:none;
background-color:rgba(0,0,0,100);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_rgb_set">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>111</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0);
border:none;
color:rgb(255,255,255);</string>
        </property>
        <property name="text">
         <string>Farbe
einstellen</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_lampen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>276</y>
     <width>181</width>
     <height>174</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">margin-top:0.5em;
border:1px solid black;
border-radius:10px;
font-size:20px;</string>
   </property>
   <property name="title">
    <string>Deckenlampen</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">;margin-top:0;
border:none;
background-color:rgba(0,0,0,100);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>111</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0);
border:none;
color:rgb(255,255,255);</string>
        </property>
        <property name="text">
         <string>An</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>groupBox_RGB</zorder>
  <zorder>groupBox_lampen</zorder>
 </widget>
 <resources>
  <include location="bilder.qrc"/>
 </resources>
 <connections/>
</ui>
